lektor/imagetools.py

```diff
@@ -332,7 +332,7 @@ def get_image_info(fp):
     if any(map(head.strip().startswith, magic_bytes)):
         return get_svg_info(fp)
 
-    _type = filetype.image(bytearray(head))
+    _type = filetype.image_match(bytearray(head))
```

Comment:
Another option here would be:
```
_type = filetype.match(bytearray(head), filetype.image_matchers)
```
This would have the advantage of being compatible with all versions of `filetype` and so wouldn't require the version pin.  (Note that `filetype.image`/`filetype.image_match` is a one-line function which does exactly this.)

Reply:
I know, but that would be giving up an undocumented feature for an even lower-level undocumented feature.

My first choice was doing a `try / import as / except ImportError`. But there's really no point in that, the very few people that do a pip install --upgrade will be served by the version pin, and those that somehow manage to install an old version will get bitten by this regardless, and there's nothing we can do about it now.